nixos/modules/services/misc/zoneminder.nix

```
@@ -289,11 +285,9 @@ in {
       phpfpm = lib.mkIf useNginx {
         pools.zoneminder = {
           inherit user group;
+          phpPackage = pkgs.php.withExtensions (e: pkgs.php.enabledExtensions ++ [ e.apcu ]);
```

I'm pretty sure that this will fail on current master, we did another change in #85026 :wink: 
```suggestion
          phpPackage = pkgs.php.withExtensions ({ enabled, all }: enabled ++ [ all.apcu ]);
```

This should work with the updated `withExtensions`, so a rebase on master and this fix and it should be all fine :slightly_smiling_face: